hoomd/data/syncedlist.py

```diff
@@ -192,7 +192,10 @@ def _register_item(self, value):
             return
         else:
             if value._attached:
-                raise RuntimeError(f"Cannot place {value} into two simulations.")
+                if not value._two_attach:
+                    raise RuntimeError(f"Cannot place {value} into two simulations.")
+                else:
+                    value._two_attach = False
```

Comment:
Why do we need this change?

I don't understand how an object can be attached to two simulations at once. They can only store one `m_sysdef`.